docs/index.md

```diff
@@ -1,25 +1,19 @@
 # Ubuntu Pro for WSL (UP4W)
-```{warning}
-This documentation describes a future release of Ubuntu Pro for WSL. Ubuntu Pro for WSL is not yet generally available in the Microsoft Store.
+```{note}
+This documentation describes a future release of UP4W. UP4W is not yet generally available in the Microsoft Store.
 ```
 
-![System Landscape](./assets/up4w-systemlandscape.png)
+Ubuntu Pro for WSL (UP4W) is a powerful automation tool for managing [WSL](https://ubuntu.com/desktop/wsl) instances from a Windows host. If you are responsible for a fleet of devices, UP4W will supercharge your ability to monitor, customise and secure WSL instances.
```

Comment:
I think we should at least once be explicit that this product cannot cover all kinds of WSL instances, but only those running Ubuntu. So, I'd rewrite this paragraph as:

```suggestion
Ubuntu Pro for WSL (UP4W) is a powerful automation tool for managing [Ubuntu WSL](https://ubuntu.com/desktop/wsl) instances from a Windows host. If you are responsible for a fleet of devices, UP4W will supercharge your ability to monitor, customise and secure WSL instances.
```

Reply:
Agreed completely! Will make this change.